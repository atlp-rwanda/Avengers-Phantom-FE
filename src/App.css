@import url("https://fonts.googleapis.com/css?family=Muli&display=swap");
@import url("https://fonts.googleapis.com/css?family=Quicksand&display=swap");

/* ::-webkit-scrollbar {
  display: none;
} */

body {
  font-family: "Muli", sans-serif;
  color: rgba(0, 0, 0, 0.8);
  font-weight: 400;
  line-height: 1.58;
  letter-spacing: -0.003em;
  font-size: 15px;
  padding: 70px;

  /* overflow: hidden; */
}

html {
  scroll-behavior: smooth;
}

::-webkit-scrollbar {
  display: none;
}

.leaflet-container {
  width: 100vw;
  height: 100vh;
}
