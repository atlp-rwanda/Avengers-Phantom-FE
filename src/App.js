import { ImportantDevices } from "@mui/icons-material";
import React from "react";
import { BrowserRouter } from "react-router-dom";
import PhantomRoutes from "./routes";
import skeletonElement from "./skeleton/SkeletonElement.js";

const App = () => {

    return (

        <BrowserRouter>
            <PhantomRoutes />
        </BrowserRouter>
    );
}

export default App
