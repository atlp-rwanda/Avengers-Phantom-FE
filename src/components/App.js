import React from "react";

const App = () => {
  return (
    <div>
      <h1>Avengers phantom</h1>
      <p>Hello Avengers FE</p>
    </div>
  );
}



export default App;
