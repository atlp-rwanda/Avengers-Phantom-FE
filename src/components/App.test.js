import React from 'react';
import { shallow } from "enzyme";
import  App  from "./App";
import toJson from 'enzyme-to-json'

describe('Test App Entry point', function () {
    it('should render app',()=>{
        shallow(<App/>);
    })
    it('should have a header tag with Avengers phantom',  ()=> {
        const wrapper = shallow(<App/>);
        expect(toJson(wrapper)).toMatchSnapshot()
    })
});