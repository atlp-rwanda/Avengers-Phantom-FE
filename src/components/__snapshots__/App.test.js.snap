// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App Entry point should have a header tag with Avengers phantom 1`] = `
<div>
  <h1>
    Avengers phantom
  </h1>
  <p>
    Hello Avengers FE
  </p>
</div>
`;
